File: trunk/hdr/C#/SnarlInterface test project/QuickStartGuide_CSharp/QuickStartGuide_CSharp/QuickStartGuide_CSharp/Form1.cs
using System;
using System.Windows.Forms;

using Snarl.V41;


namespace QuickStartGuide_CSharp
{
	public partial class Form1 : Form
	{
		private SnarlConnector snarl = new SnarlConnector();

		public Form1()
		{
			InitializeComponent();
		}

		private void button1_Click(object sender, EventArgs e)
		{
			snarl.RegisterApp("QuickStartGuide_CSharp", "QuickStartGuide_CSharp", "", IntPtr.Zero, 0, 0);
		}

		private void button2_Click(object sender, EventArgs e)
		{
			snarl.EZNotify("MsgId", "Title", "Text message", 10, "", 0, "", "");
		}

		private void Form1_FormClosed(object sender, FormClosedEventArgs e)
		{
			snarl.UnregisterApp();
		}
	}
}

File: hdr/C#/SnarlInterface test project/QuickStartGuide_VBNet/SnarlHowTo_VBNet/Form1.vb
Imports Snarl


Public Class Form1

	Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
		Dim nReturnId As Integer
		nReturnId = SnarlConnector.ShowMessage("Test Title", "Test Text with 10s timeout", 10, "", 0, 0)
	End Sub
End Class
